deployments/run.sh

```diff
@@ -42,7 +42,7 @@ function show_help () {
   exit 1
 }
 
-RIALTO=' -f ./networks/rialto.yml'
+RIALTO=' -f ./networks/rialto.yml -f ./networks/rialto-parachain.yml'
```

Comment:
Since we don't have any parachain<>parachain || parachain<>relaychain bridges now, parachain now will be started after main Rialto relaychain. In future we may start this only if P2P || P2R bridge is started.